air/src/aux_table/bitwise_pow2/tests.rs

```diff
@@ -0,0 +1,52 @@
+use super::super::bitwise_pow2;
+use super::bitwise::tests::get_test_frame_with_two_selectors;
+use super::{bitwise, pow2};
+use crate::{Felt, FieldElement};
+use vm_core::bitwise::{BITWISE_OR, BITWISE_XOR};
+
+// UNIT TESTS
+// ================================================================================================
+
+/// Total constraints in bitwise_pow2 constraint. Sum of bitwise_pow2 aux_table constraint(4) +
+/// bitwise(14) + pow2(24)
```

Comment:
This comment should also be updated to remove the numbers, now that the the constant is not being hardcoded.